app/views/templates/run_command.erb

```diff
@@ -1,8 +1,8 @@
 <%#
 kind: job_template
 name: Run Command - SSH Default
-job_name: Run Command
-description_format: "%{job_name} %{command}"
+job_category: Commands
+description_format: "%{job_category} %{command}"
```

Comment:
I think this needs a better description, because now it renders `Commands hostname -f`

Maybe something simple like `Run %{command}`